tests/report_status_test.go

```diff
@@ -59,5 +68,11 @@ var _ = Describe("Reporting State", func() {
 			Expect(nodeNetworkState.Spec.NodeName).To(Equal(nodeName))
 		})
 
+		It("should have a cni0 bridge with all the veth interfaces", func() {
```

Comment:
Please add a TODO to drop this once we have test coverage for a bridge created by us, `cni0` just happens to be there

Reply:
I think this test will only pass on k8s with flannel

Reply:
We could add some code creating our own bridge on the host and testing that, but I think that would be a waste of time. The next step is testing our demos - we will get to check this on bridges created by us.

Reply:
We can drop the functional test part of this review implement the linux bridge demo and test it there 
what do you think ?

Reply:
Ok I have mutate the PR into the linux bridge demo, since it's the most importnat thing for this project now.